sdk/core/azure-core/inc/azure/core/http/websockets/websockets.hpp

```diff
@@ -311,13 +311,12 @@ namespace Azure { namespace Core { namespace Http { namespace WebSockets {
        */
       void Open(Azure::Core::Context const& context = Azure::Core::Context{});
 
-      /** @brief Closes a WebSocket connection to the remote server gracefully.
-       *
-       * @param context Context for the operation.
+      /** @brief Closes a WebSocket connection to the remote server.
        */
-      void Close(Azure::Core::Context const& context = Azure::Core::Context{});
+      void Close();
```

Comment:
This is `_internal` namespace, correct?

Even if it is `_internal` previously released Stable versions should not be failing when `Core` gets updated to the newest version. Would it be so? If not, leave the argument but ignore it.

Reply:
Currently WebSockets is not _internal. However, it's unreleased (it's in feature/websockets to allow for modifications to the API surface).